src/reducers/places.js

```diff
@@ -0,0 +1,46 @@
+import { ADD_PLACE } from '../actions/index'
+
+const mockedState =[{
+  name: 'Prague Castle',
+  day: 1,
+  category: 'Sightseeing',
+  placeId: 'ChIJw7ckbB6VC0cRnyUSr4g8zyo',
+  long: 14.401616500000046,
+  lat: 50.0910966
+}, {
+  name: 'Prague Old Town',
+  day: 1,
+  category: 'Sightseeing',
+  placeId: 'ChIJ6dXlGemUC0cRUfsgXBCvACY',
+  long: 14.419407399999955,
+  lat: 50.0894377
+}, {
+  name: 'Pizza Nuova',
+  day: 1,
+  category: 'Eating Out',
+  placeId: 'ChIJCZlGiuqUC0cRbI2NYf9dwzo',
+  long: 14.427775600000018,
+  lat: 50.08944389999999
+}]
+
+const places = (state = mockedState, action) => {
+  switch (action.type) {
+    case ADD_PLACE:
+      return [
+        ...state,
+        {
+          name: action.name,
+          day: action.day,
+          category: action.category,
+          placeId: action.placeId,
+          long: action.long,
+          lat: action.lat
```

Comment:
Widziałem, że często używa się `action.payload`, żeby nie przepisywać wszystkich wartości, ale w sumie nie używasz Flow/TS, więc może tak bezpieczniej...

Reply:
moze bede potem dodawac TS, bo chcialbym sie tez nauczyc, ale to dopiero jak sie poczuje komfortowo z css in js i reduxem

Reply:
TS jest super (w większości :P) i ogólnie to nie jest aż tak trudny. Zwłaszcza jak masz doświadczenie z Javą to będzie pikuś. Największy problem to i guess skonfigurowanie :/ Bo to osobny linter i compiler i połączyć z webpackiem i babelem i omg :P ale da się :P

Reply:
I też w sumie z TS czy Flow to jest tak, że jak chcesz to trzeba iść all in. Bo jak część masz safe część unsafe to w sumie wszystko masz do dupy :P I nie czerpiesz benefitów z TS a tylko masz errory i dodatkową pracę. Chyba, że masz jakieś odizolowane części aplikacji to spoko